integtest/spec/all_books_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+RSpec.describe 'building all books' do
+  context 'for a minimal config' do
+    convert_all_before_context do |src|
+      src.write 'source/index.asciidoc', <<~ASCIIDOC
+        = Title
+
+        == Chapter
+
+        Some text.
+      ASCIIDOC
+      src.init_repo 'source'
+      src.write 'conf.yaml', <<~YAML
```

Comment:
I think I'll end up having to make generating this file more parameterized and automatic but I'd like to wait for a follow up for that.